client/extensions/woocommerce/app/store-stats/utils.js

```diff
@@ -1,12 +1,12 @@
-/** @format */
 /**
  * External dependencies
  */
 import { find, includes, forEach, findIndex, round } from 'lodash';
 import classnames from 'classnames';
-import { moment, translate, numberFormat } from 'i18n-calypso';
+import { translate, numberFormat } from 'i18n-calypso';
 import qs from 'qs';
 import formatCurrency from '@automattic/format-currency';
+import moment from 'moment'; // No localization needed in this file.
```

Comment:
Please double-check this is true.

Reply:
I looked at the functions in this file and It doesn't look like they need to be localized.